sdk/java-sdk/src/main/java/org/lightswitch/sdk/model/FeatureFlag.java

```diff
@@ -0,0 +1,56 @@
+package org.lightswitch.sdk.model;
+
+import org.lightswitch.sdk.user.LightSwitchUser;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class FeatureFlag {
+
+    private final String key;
+    private final Object defaultValue;
+    private final Map<Map<String, String>, Object> filters; // filtering rules
```

Comment:
Now that the `filters` have been added to flag, it seems like similar modifications should be needed in the backend entity and API spec as well. It would be good to review and align everything before moving forward.